lib/white_bread/outputers/html.ex

```diff
@@ -0,0 +1,75 @@
+defmodule WhiteBread.Outputers.HTML do
+  use GenServer
+  alias WhiteBread.Gherkin.Elements.Scenario
+  alias WhiteBread.Gherkin.Elements.ScenarioOutline
+  alias WhiteBread.Outputers.HTML.Formatter
+
+  @moduledoc """
+  This generic server accumulates information about White Bread
+  scenarios then formats them as HTML and outputs them to a file in
+  one go.
+  """
+
+  defstruct pid: nil, path: nil, data: []
+
+  ## Client Interface
+
+  def start do
+    {:ok, outputer} = GenServer.start __MODULE__, []
+    %__MODULE__{pid: outputer}
+  end
+
+  def stop(%__MODULE__{pid: outputer}) do
+    :ok = GenServer.stop outputer, :normal, 2 * 1000
+  end
+
+  def report(%__MODULE__{pid: outputer}, report) do
+    GenServer.cast outputer, report
+  end
+
+  ## Interface to Generic Server Machinery
+
+  def init(_) do
+    {:ok, %__MODULE__{path: path()}}
+  end
+
+  def handle_cast({:scenario_result, {result, _}, %Scenario{name: name}}, state) when :ok == result or :failed == result do
+    {:noreply, %{ state | data: [ {result, name} | state.data ]}}
+  end
+
```

Comment:
Convention seems different in Elixir: https://github.com/elixir-lang/elixir/blob/master/lib/ex_unit/lib/ex_unit/server.ex#L59-L67